components/CardDetails.tsx

```diff
@@ -0,0 +1,74 @@
+'use client';
+
+import React, { Fragment } from 'react';
+import Image from 'next/image';
+
+import { Dialog, Transition } from '@headlessui/react';
+
+interface CardDetailsProps {
+  isOpen: boolean;
+  closeModal: () => void;
+}
+
+const CardDetails = ({ isOpen, closeModal }: CardDetailsProps) => {
```

Comment:
Might be a stylist preference but I'd do:
```js
export default function CardDetails({ isOpen, closeModal }: CardDetailsProps) {
```

Reply:
Could you please explain why to change from an arrow function to a function declaration in this case? Should we reformat everything to keep the same semantic style? Changes has been applied and waiting for your comments

Reply:
Because like this you can declare the component and do a default export in the same line

Export default function MyComponent

Otherwise you have to do it in two lines

Const MyComponent
...
Export default MyComponent